src/components/vive-controls.js

```diff
@@ -105,6 +106,7 @@ module.exports.Component = registerComponent('vive-controls', {
     el.removeEventListener('touchstart', this.onButtonTouchStart);
     el.removeEventListener('model-loaded', this.onModelLoaded);
     el.removeEventListener('axismove', this.onAxisMoved);
+    this.controlllerEventsActive = false;
```

Comment:
We need to call `addEventListener` in the `play` handler only when coming out of a pause. We don't want to call it directly because `utils.tracked-controls` will already call it and we don't want to call it on the first initialization.